components/brave_new_tab_ui/components/search/SearchContext.tsx

```diff
@@ -73,13 +73,15 @@ class SearchPage implements PageInterface {
     for (const listener of this.selectionListeners) listener(selection)
   }
 
-  setInputText(inputText: string) {}
-  setThumbnail(thumbnailUrl: string) {}
+  setInputText(inputText: string) { }
+  setThumbnail(thumbnailUrl: string) { }
 }
 
 export const search = new SearchPage()
 
 export function SearchContext(props: React.PropsWithChildren<{}>) {
+  const { isSearchEngineEnabled } = useEnabledSearchEngines()
+  const { lastNtpSearchEngine, setLastNtpSearchEngine } = useNtpWidgetDefaultSearchEngine()
```

Comment:
If we're exposing these as hooks, we should make it possible to use them independently of the context (i.e. consumers should be able to use these hooks directly)

Reply:
I don't think these hooks are necessarily tied to the context, but it made sense to have them be used in the context as that is doing most of the heavy lifting for changing the search engine